features/promotion/managing_coupons/browsing_coupon.feature

```diff
@@ -0,0 +1,16 @@
+@managing_promotion_coupons
+Feature: Browsing promotion coupons
+    In order to see all promotion coupons
+    As an Administrator
+    I want to browse coupons
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has promotion "Christmas sale" with coupon "Santa's gift"
```

Comment:
Santa's gift does not sound like a real coupon name, should be smth like: ``SANTA2016``.